### PR TITLE
core, editoast: remove schematic

### LINKED ISSUE #6789: Delete schematic geometry from the application
## Description

The schematic is already removed from the front end. We don't want to handle it in the backend.

## What should be done

- Create a migration that alters layer tables (drop schematic columns)
- Adapt railjson to remove the schematic from track sections (bump version)
  - Adapt core
  - Adapt railjson-generator
  - Adapt connectors
- Adapt editoast layer generation
- Adapt map view configuration
- Adapt InfraCache

### PR DESCRIPTION
**Pull request by Khoyo**
Fixes #6789.

### CODE REVIEW

File: editoast/editoast_schemas/src/primitives/zone.rs [hunk 1]
```diff
@@ -7,17 +7,15 @@ editoast_common::schemas! {
...
 }
 
-/// Geographic and Schematic bounding box zone impacted by a list of operations.
+/// Geographic bounding box zone impacted by a list of operations.
 /// Zones use the coordinate system [epsg:4326](https://epsg.io/4326).
 #[derive(Debug, Clone, Default, Serialize, ToSchema)]
 pub struct Zone {
     pub geo: BoundingBox,
-    pub sch: BoundingBox,
 }
```
Comment (flomonster) [inline editoast/editoast_schemas/src/primitives/zone.rs]: I don't think we need the `Zone` struct anymore.

**Review by flomonster — APPROVED** [review]

File: front/src/reducers/osrdconf/infra_schema.json [hunk 2]
```diff
@@ -1593,7 +1593,7 @@
             "type": "object"
         },
         "TrackSection": {
-            "description": "A track section is a piece of track and is the tracking system used in OSRD to locate a point.\nA track section is identified by his unique id and its coordinates (geographic or schematic).",
```
Comment (clarani) [inline front/src/reducers/osrdconf/infra_schema.json:1596]: Can you also update the french translation ? You can find it in public/locales/fr/infraEditor.json ? You can search for "schématiques"

**Review by clarani — COMMENTED** [review]
Almost good, I found 2 more occurrences of sch or schematic:
- in injectGeometry (trackEdition/utils.ts)
- in loadDataModel (editor/thunkActions.ts)

**Review by eckter — APPROVED** [review]
LGTM for core and tests

**Review by clarani — APPROVED** [review]
LGTM :white_check_mark:

**Pull request merged**